packages/apps/graph/src/scripts/devnet/crosschain-transfer.ts

```diff
@@ -93,9 +93,11 @@ export async function crossChainTransfer({
   const status = await listen(submittedTx);
   inspect('Transfer Result')(status);
   if (status.result.status === 'failure') {
-    throw new Error('DEBIT REJECTED');
+    throw new Error('Transfer failed');
   }
 
+  const originalConsoleLog = console.log;
+  console.log = () => {};
```

Comment:
Is this so the spv log doesn't show up anymore? I'd rather have it show up than overwriting system functions

Reply:
This was a mistake, shouldn't be there. I'll remove it. The spv log will be replaced in the client itself and we will not overwrite system functions